includes/admin/views/html-admin-settings.php

```diff
@@ -107,18 +107,8 @@
              */
             if (Utils::isPluginActive('give-recurring/give-recurring.php')) {
                 echo '
-                         <a class="give-nav-addons-tab" href="https://docs.givewp.com/settings-star" target="_blank">
-                            <svg width="22" height="22" viewBox="0 0 22 22" fill="none" xmlns="http://www.w3.org/2000/svg">
-                                <path
-                                    d="M4.5 1C4.5 0.447715 4.05228 0 3.5 0C2.94772 0 2.5 0.447715 2.5 1V2.5H1C0.447715 2.5 0 2.94772 0 3.5C0 4.05228 0.447715 4.5 1 4.5H2.5V6C2.5 6.55228 2.94772 7 3.5 7C4.05228 7 4.5 6.55228 4.5 6V4.5H6C6.55228 4.5 7 4.05228 7 3.5C7 2.94772 6.55228 2.5 6 2.5H4.5V1Z"
-                                    fill="#0C7FF2" />
-                                <path
-                                    d="M4.5 16C4.5 15.4477 4.05228 15 3.5 15C2.94772 15 2.5 15.4477 2.5 16V17.5H1C0.447715 17.5 0 17.9477 0 18.5C0 19.0523 0.447715 19.5 1 19.5H2.5V21C2.5 21.5523 2.94772 22 3.5 22C4.05228 22 4.5 21.5523 4.5 21V19.5H6C6.55228 19.5 7 19.0523 7 18.5C7 17.9477 6.55228 17.5 6 17.5H4.5V16Z"
-                                    fill="#0C7FF2" />
-                                <path
-                                    d="M12.9333 1.64102C12.7848 1.25483 12.4138 1 12 1C11.5862 1 11.2152 1.25483 11.0667 1.64102L9.33248 6.14988C9.03207 6.93093 8.93768 7.156 8.80855 7.33759C8.67899 7.5198 8.5198 7.67899 8.33759 7.80855C8.156 7.93768 7.93093 8.03207 7.14988 8.33248L2.64102 10.0667C2.25483 10.2152 2 10.5862 2 11C2 11.4138 2.25483 11.7848 2.64102 11.9333L7.14988 13.6675C7.93093 13.9679 8.156 14.0623 8.33759 14.1914C8.5198 14.321 8.67899 14.4802 8.80855 14.6624C8.93768 14.844 9.03207 15.0691 9.33248 15.8501L11.0667 20.359C11.2152 20.7452 11.5862 21 12 21C12.4138 21 12.7848 20.7452 12.9333 20.359L14.6675 15.8501C14.9679 15.0691 15.0623 14.844 15.1914 14.6624C15.321 14.4802 15.4802 14.321 15.6624 14.1914C15.844 14.0623 16.0691 13.9679 16.8501 13.6675L21.359 11.9333C21.7452 11.7848 22 11.4138 22 11C22 10.5862 21.7452 10.2152 21.359 10.0667L16.8501 8.33248C16.0691 8.03207 15.844 7.93768 15.6624 7.80855C15.4802 7.67899 15.321 7.5198 15.1914 7.33759C15.0623 7.156 14.9679 6.93093 14.6675 6.14988L12.9333 1.64102Z"
-                                    fill="#0C7FF2" />
-                            </svg>
+                         <a class="give-nav-addons-tab" href="edit.php?post_type=give_forms&page=give-add-ons" >
```

Comment:
@JoshuaHungDinh I recommend using `admin_url()` instead of relative paths. The reason is it helps make sure the URLs work properly on complex WP sites, especially multi-sites.

Reply:
Ok, got it! Resolved: https://github.com/impress-org/givewp/pull/6794/commits/d2270f81e783191e37f1233f56ba16496bef63f4  https://github.com/impress-org/givewp/pull/6794/commits/7915c3799d303aabf0dba7b518f09d9d3852aab7